File: app/src/main/java/konradrutkowski/com/tapapp/fsquare/Place.java
package konradrutkowski.com.tapapp.fsquare;

import java.io.Serializable;

public class Place implements Serializable {

    private String url;
    private String ID;
    private String name;
    private String city;
    private String category;
    private String checkins;
    private String distance;
    private String address;
    private long dbID;

    public long getDbID() {
        return dbID;
    }

    public void setDbID(long dbID) {
        this.dbID = dbID;
    }

    public String getUrl() {
        return url;
    }

    public String getID() {
        return ID;
    }

    public String getName() {
        return name;
    }


    public String getCity() {
        return city;
    }

    public String getCategory() {
        return category;
    }

    public String getCheckins() {
        return checkins;
    }

    public String getDistance() {
        return distance;
    }

    public String getAddress() {
        return address;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public void setID(String ID) {
        this.ID = ID;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public void setCheckins(String checkins) {
        this.checkins = checkins;
    }

    public void setDistance(String distance) {
        this.distance = distance;
    }

    public void setAddress(String address) {
        this.address = address;
    }
}

File: app/src/main/java/konradrutkowski/com/tapapp/fsquare/Place.kt
package konradrutkowski.com.tapapp.fsquare

import android.os.Parcel
import android.os.Parcelable

class Place() : Parcelable {
    var url: String? = null
    var id: String? = null
    var name: String? = null
    var city: String? = null
    var category: String? = null
    var checkins: String? = null
    var distance: String? = null
    var address: String? = null
    var dbID: Long = 0

    constructor(parcel: Parcel) : this() {
        url = parcel.readString()
        id = parcel.readString()
        name = parcel.readString()
        city = parcel.readString()
        category = parcel.readString()
        checkins = parcel.readString()
        distance = parcel.readString()
        address = parcel.readString()
        dbID = parcel.readLong()
    }

    override fun writeToParcel(parcel: Parcel, flags: Int) {
        parcel.writeString(url)
        parcel.writeString(id)
        parcel.writeString(name)
        parcel.writeString(city)
        parcel.writeString(category)
        parcel.writeString(checkins)
        parcel.writeString(distance)
        parcel.writeString(address)
        parcel.writeLong(dbID)
    }

    override fun describeContents(): Int {
        return 0
    }

    companion object CREATOR : Parcelable.Creator<Place> {
        override fun createFromParcel(parcel: Parcel): Place {
            return Place(parcel)
        }

        override fun newArray(size: Int): Array<Place?> {
            return arrayOfNulls(size)
        }
    }
}


